core/src/main/java/com/linecorp/armeria/unsafe/package-info.java

```diff
@@ -13,14 +13,14 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-
 /**
  * Utilities for working with {@link io.netty.buffer.ByteBuf} in an unsafe way. These can improve performance
  * when dealing with large buffers but require careful memory management or there will be memory leaks. Only use
  * these methods if you really know what you're doing.
  */
-
+@UnstableApi
```

Comment:
Do we also need to add to the classes? (I noticed we did in `grpc-protocol`)

Reply:
I did already.

Reply:
Oops not sure why I missed it before